src/blueprints/tv2/action-factories/tv2-split-screen-action-factory.ts

```diff
@@ -357,56 +357,56 @@ export class Tv2DveActionFactory {
+        this.videoMixerTimelineObjectFactory.createSplitScreenBoxesTimelineObject(boxes, PLANNED_SPLIT_SCREEN_TIMELINE_OBJECT_PRIORITY),
+        this.videoMixerTimelineObjectFactory.createSplitScreenPropertiesTimelineObject(blueprintConfiguration, splitScreenConfiguration.layoutProperties),
+        this.videoMixerTimelineObjectFactory.createProgramTimelineObject(splitScreenSource, videoSwitcherTimelineEnable),
+        this.videoMixerTimelineObjectFactory.createCleanFeedTimelineObject(splitScreenSource, videoSwitcherTimelineEnable),
+        this.videoMixerTimelineObjectFactory.createLookaheadTimelineObject(splitScreenSource, videoSwitcherTimelineEnable),
+        this.casparCgTimelineObjectFactory.createSplitScreenKeyTimelineObject(this.assetFolderHelper.joinAssetToFolder(blueprintConfiguration.studio.DVEFolder, splitScreenConfiguration.key)),
+        this.casparCgTimelineObjectFactory.createSplitScreenFrameTimelineObject(this.assetFolderHelper.joinAssetToFolder(blueprintConfiguration.studio.DVEFolder, splitScreenConfiguration.frame)),
+        this.casparCgTimelineObjectFactory.createSplitScreenLocatorTimelineObject(),
         ...audioTimelineObjects
       ]
 
       return {
-        id: `plannedDveAsNextAction_${data.name.replace(/\s/g, '')}`,
+        id: `plannedSplitScreenAsNextAction_${data.name.replace(/\s/g, '')}`,
         name: data.template,
         description: '',
         type: PartActionType.INSERT_PART_AS_NEXT,
         metadata: {
           contentType: Tv2ActionContentType.SPLIT_SCREEN
         },
         data: {
-          partInterface: this.createPartInterface(partId, dveConfiguration),
-          pieceInterfaces: [this.createDvePieceInterface(partId, dveConfiguration.name, metadata, dveTimelineObjects)]
+          partInterface: this.createPartInterface(partId, splitScreenConfiguration),
+          pieceInterfaces: [this.createSplitScreenPieceInterface(partId, splitScreenConfiguration.name, metadata, splitScreenTimelineObjects)]
         }
       }
     })
   }
 
-  private mapDveBoxInputToNumber(dveBoxInput: DveBoxInput): number {
-    switch (dveBoxInput) {
-      case DveBoxInput.INPUT_1: {
+  private mapSplitScreenBoxInputToNumber(splitScreenBoxInput: SplitScreenBoxInput): number {
+    switch (splitScreenBoxInput) {
+      case SplitScreenBoxInput.INPUT_1: {
         return 0
       }
-      case DveBoxInput.INPUT_2: {
+      case SplitScreenBoxInput.INPUT_2: {
         return 1
       }
-      case DveBoxInput.INPUT_3: {
+      case SplitScreenBoxInput.INPUT_3: {
         return 2
       }
-      case DveBoxInput.INPUT_4: {
+      case SplitScreenBoxInput.INPUT_4: {
         return 3
       }
     }
   }
 
-  private createRecallLastDveAction(): Tv2RecallDveAction {
+  private createRecallLastSplitScreenAction(): Tv2RecallSplitScreenAction {
     return {
-      id: 'recall_last_dve_action',
+      id: 'recall_last_split_screen_action',
       name: 'Recall DVE',
```

Comment:
So i'm assuming this is intention, but why would we not scrub the entirety out even though it is in the blueprints?

Reply:
I am assuming you are referring to the `name` property?
If that is the case then, we don't want to rename it, since it is a user facing value, where it should be DVE.

Reply:
I can see the argument if they are user facing strings :-)